code/game/gamemodes/vampire/vampire.dm

```diff
@@ -182,7 +182,7 @@
 	if (you_are)
 		dat = "<B>\red You are a Vampire! \black</br></B>"
 	dat += {"To bite someone, target the head and use harm intent with an empty hand. Drink blood to gain new powers.
-You are weak to holy things and starlight. Don't go into space and avoid the Chaplain, the chapel and especially Holy Water."}
+You are weak to holy things. Avoid the Chaplain, the chapel and Holy Water."}
```

Comment:
please return the original text so people aren't asking "why am I burning in space?"